reports/default/json-schema.json

```diff
@@ -6894,6 +6903,96 @@
                     "minimum": 0
                 }
             }
+        },
+        "market_prices": {
+            "type": "object",
+            "additionalProperties": false,
+            "description": "Определение стоимости ТС",
+            "properties": {
+                "ads": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                        "items": {
+                            "type": "array",
+                            "items": {
+                                "type": "object",
+                                "additionalProperties": false,
+                                "properties": {
+                                    "amount": {
+                                        "type": "object",
+                                        "additionalProperties": false,
+                                        "properties": {
+                                            "max": {
+                                                "description": "Максимальное значение стоимости ТС, рассчитанное на основе объявлений о продаже ТС",
+                                                "type": [
+                                                    "integer",
+                                                    "null"
+                                                ],
+                                                "examples": [
+                                                    630000
+                                                ],
+                                                "fillable_by": [
+                                                    "ads.price"
+                                                ]
+                                            },
```

Comment:
this value can be negative? if not - we should set minimal value to `0`

Reply:
This recommendation can be applied to the all `integer` properties

Reply:
Fix